test/executable/AxelarExecutable.js

```diff
@@ -11,13 +11,13 @@ const {
 } = ethers;
 const getRandomID = () => id(Math.floor(Math.random() * 1e10).toString());
 
-describe('GMPExecutable', () => {
+describe('AxelarExecutable', () => {
     let gatewayFactory;
 
     let destinationChainGateway;
 
-    let GMPExecutableFactory;
-    let GMPExecutable;
+    let AxelarExecutableFactory;
+    let AxelarExecutable;
```

Comment:
```suggestion
    let executable;
```
since this is a normal variable and not a class or import

Reply:
agreed. updated.